src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,55 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
```

Comment:
The class name `FruitTransaction` is not descriptive enough. It would be better to name it `Transaction` because it's not just fruit-specific.